interface-definitions/system-logs.xml.in

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interfaceDefinition>
+  <node name="system">
+    <children>
+      <node name="logs" owner="${vyos_conf_scripts_dir}/system-logs.py">
+        <properties>
+          <help>Logging options</help>
+          <priority>9999</priority>
+        </properties>
+        <children>
+          <node name="logrotate">
+            <properties>
+              <help>Logrotate options</help>
+            </properties>
+            <children>
+              <node name="atop">
+                <properties>
+                  <help>Atop logs options</help>
+                </properties>
+                <children>
+                  <leafNode name="maxsize">
+                    <properties>
+                      <help>Size of a single log file that triggers rotation</help>
+                      <valueHelp>
+                        <format>u32:1-1024</format>
+                        <description>Size in MB</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="numeric" argument="--range 1-1024" />
+                      </constraint>
+                      <constraintErrorMessage>The size must be between 1 and 1024 MB</constraintErrorMessage>
+                    </properties>
+                    <defaultValue>10</defaultValue>
+                  </leafNode>
+                  <leafNode name="rotate">
+                    <properties>
+                      <help>Count of rotations before old logs will be deleted</help>
+                      <valueHelp>
+                        <format>u32:1-100</format>
+                        <description>Rotations</description>
```

Comment:
Please add a (default: 10) to the end of the description so the user knows it.

Reply:
Done.